addons/binding/org.openhab.binding.zwave/src/main/java/org/openhab/binding/zwave/internal/protocol/serialmessage/AddNodeMessageClass.java

```diff
@@ -67,9 +79,12 @@ public SerialMessage doRequestStop() {
         SerialMessage newMessage = new SerialMessage(SerialMessage.SerialMessageClass.AddNodeToNetwork,
                 SerialMessage.SerialMessageType.Request, SerialMessage.SerialMessageClass.AddNodeToNetwork,
                 SerialMessage.SerialMessagePriority.High);
-        byte[] newPayload = { (byte) ADD_NODE_STOP };
 
-        newMessage.setMessagePayload(newPayload);
+        ByteArrayOutputStream outputData = new ByteArrayOutputStream();
+        outputData.write(ADD_NODE_STOP);
+        outputData.write(255); // TODO: This should use the callbackId
```

Comment:
It’s intended - it’s the callback id and it fixes a bug with the incorrectly formatted packet.